content/source/docs/cloud/overview.html.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: "cloud"
+page_title: "Overview of Features - Terraform Cloud"
+---
+
+# Overview of Terraform Cloud Features
+
+[cli]: /docs/cli-index.html
+[remote backend]: /docs/backends/types/remote.html
+[speculative plans]: ./run/index.html#speculative-plans
+[remote_state]: /docs/providers/terraform/d/remote_state.html
+[outputs]: /docs/configuration/outputs.html
+[modules]: /docs/modules/index.html
+[terraform enterprise]: /docs/enterprise/index.html
+
+
+Terraform Cloud is an execution platform that runs Terraform. Unlike a general-purpose execution platform, it has deep integrations with Terraform's workflows and data formats, which it uses to make Terraform more convenient and more powerful.
```

Comment:
So, this is part of why I went on for a while about the intended audience in the PR description! That marketing quote is technically correct, and I'll stipulate that it's genuinely useful for someone operating at one or two removes from the process of actually using Terraform. But for a direct practitioner, it's obscuring instead of clarifying, because it doesn't say what TFC actually DOES -- to wit, it's a Terraform-specific CI system ("execution platform") that does things that would be impossible or incredibly frustrating to try and build yourself using an off-the-shelf CI platform. 

The constraint I'm working within is that we need at least ONE place on the site where we tell our users straight-out that TFC exists in order to execute Terraform in response to various events. Given that, I'm open to better ways to express it! Any thoughts?

Reply:
Good points. I think I'm stuck on "execution platform" - that sounds like it has a specific meaning, but I haven't see that term used to describe similar systems. Just a swag:

> Terraform Cloud (TFC) is a a Software as a Service platform that runs Terraform in a secure container environment on your behalf. It has deep integrations with Terraform's workflows and data formats, which it uses to make Terraform more convenient to use and powerful.